gossip.asciidoc

```diff
@@ -644,68 +644,388 @@ channel graph) has the following attributes:
+`msat` or milli-satoshis instead of satoshis or Bitcoin. We use `msat`, as it
+allows us to charge fees for 1 sat payments. The routing fees has two
+component, `fee_base_msat` and `fee_proportional_millionths`. In order to
+calculate how much a payment of `N msat` should pay to route through a channel,
+we use the following equation:
+```
+fee = fee_base_msat + (N * fee_proportional_millionths) / 1,000,000
+```
 
-// TODO(roasbeef): rename to "the structure of the channel graph"?
+This may look a bit strange at first, but all we're doing in the latter half of
+the equation is computing the rate to be paid. Rather than floating point
+arithmetic, we use fixed point arithmetic with 1 million as our base. This lets
+us express very small amounts of fees to be paid in the `mast` unit. Let's say
+we use a value of `100,000` for `fee_proportional_millionths`, this is
+equivalent to a `10%` fee as `100,000/1,000,000 = 0.10` or `10%.
+
+Another important security related parameter for a channel is the
+`cltv_expiry_delta` field. This controls the delta between the incoming and
+outgoing HTLCs that the routing node dictates. We refer to this as a security
+parameter, as it's the of blocks between the expiry of the outgoing HTLC and
+the incoming HTLC. After the outgoing HTLC expires, the routing node has
+`cltv_expiry_delta` blocks to fully resolve it (sweep on timeout) before the
+incoming time-lock expires. If the routing node cannot resolve the outgoing
+HTLC within `cltv_expiry_delta` blocks, then a _race condition_ arises. This
+race condition can result in a loss of funds, as after this period, _both_ the
+outgoing and incoming time locks have expired. If the party that extended the
+incoming HTLC sweeps it on chain with the timeout clause and the party that
+accepted the outgoing HTLC redeems it with a pre-image, then the forwarding
+nodes has lots funds as it only sent funds out and didn't receive any in.
+
+The remaining fields will govern the availability of the channel, as well as
+the amount of HTLCs that can be routed through the channel. The `channel_flags`
+field is a bitfield that allows the owner of a `channel_update` to _disable_
+the channel in a particular direction. This allows a node to disable routing
+through the channel, but still permits it to use the channel for its own
+payments. If the second LSB bit is `1`, then the channel is disabled, otherwise
+it's enabled.
+
+Next, we have the `htlc_minimum_msat` and `htlc_maximum_msat` fields. As their
+name entails, this allows a node operator to restrict the _size_ of HTLCs that
+are permitted to be routed through the channel. Setting these values can
+recreate a sort of "routing profile" for the channel. As an example, if we set
+our `htlc_minimum_msat` to a very high value, then we signal that this channel
+isn't to be used to route micropayments. If we set our `htlc_maximum_msat` to a
+very low value, then we signal the converse, instead that this channel is only
```

Comment:
should we discuss the issue with dustoutputs? 

we also did not discuss the `max accepted htlc` parameter. Even if it is not part of gossip it should be discussed as a security thing as other things that happen in gossip. 

Bot topics could be info boxes